ha/setup-cluster.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+set -x
+
+WORK_DIR="${1:-$(pwd)}"
+CONFIG_DIR="${2:-"${WORK_DIR}/config"}"
+
+CREATE_VM_SCRIPT="$(pwd)/create-vm.sh"
+
+VM01="node01"
```

Comment:
Let's namespace these a bit better, e.g. `VM01="fence-test-virsh-node01"`. It will be easier to remember what those VMs are about when doing `virsh list` or similar on the nodes :) Also namespacing the VMs like this allows scripts to know the VMs they "own" and can do destructive operations on (e.g. for cleanup).